designer/client/components/Flyout/Flyout.tsx

```diff
@@ -1,11 +1,13 @@
 import React, { useContext, useEffect, useLayoutEffect, useState } from "react";
 import FocusTrap from "focus-trap-react";
 import { FlyoutContext } from "../../context";
+import { DataContext } from "../../context";
```

Comment:
you can make this a bit neater by doing 
```suggestion
import { FlyoutContext, DataContext } from "../../context"
```

Reply:
Getting merge conflicts for some reason